apps/ui/components/EditUserModal.vue

```diff
@@ -182,15 +179,14 @@ async function saveUser() {
 
   await updateUser(editedUser);
   emit("userUpdated");
-  modal.value?.close();
+  visible.value = false;
 }
 
 function closeModal() {
-  modal.value?.close();
+  visible.value = false;
 }
 
-defineExpose({
-  show: showModal,
-  close: closeModal,
-});
+const visible = ref(false);
+
+
```

Comment:
I think the formatter in the ui app is skipping files, as it should remove these enters.